CHANGES.md

```diff
@@ -1,11 +1,300 @@
 Release Changes List
 ====================
 
+version 1.27.0
+==============
+
+Thirtieth public release of Chapel, June 30, 2022
+
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+* improved GPU support, including enabling the use of multiple locales
+* improved namespace behavior w.r.t. `import`, `use`, and shadowing
+* the Chapel compiler now supports LLVM 14, in addition to 11, 12, and 13
+* improved the performance and scalability of scan operations on `Block` arrays
+* added new 'OS.POSIX' and 'GPUDiagnostics' modules
+* restored support for Omni-Path networks
+* introduced a Code of Conduct for Chapel developers
+* made many changes, mostly minor, in support of Chapel 2.0 stabilization
```

Comment:
Is this the first mention of 2.0 in the CHANGES? Wondering if this needs more context or if we've talked about it enough in release notes?